tendrl_node_agent/monitoring/plugins/ifspeed.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Now its no more going to be part of node_agent. IT'll be part of performance_monitoring but a separate rpm for server and node-side pieces will be carved out of it